src/main/java/HWS/HW8/Bank.java

```diff
@@ -0,0 +1,106 @@
+package HWS.HW8;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Bank {
+    private Transactions transactions;
+    int accountsQuantityLimit;
+    ArrayList<Account> accountsList = new ArrayList<>();
+    ArrayList<String> transactionsLog = new ArrayList<>();
+
+    Bank(int accountsQuantityLimit) {
+        this.accountsQuantityLimit = accountsQuantityLimit;
+    }
+
+    public Account createAccount(int number, double balance) {
+        if (accountsList.size() == accountsQuantityLimit) {
+            System.out.println("Impossible to create new account with number:" + number + ", the limit of accounts has been reached.");
+            return null;
+        }
+        if (isAccountNumberExists(number)) {
+            System.out.println("Impossible to create new account with number: " + number + ". Account with such number already exists.");
+            return null;
+        }
+        accountsList.add(new Account(number, balance));
+        return accountsList.get(accountsList.size() - 1);
```

Comment:
Not sure if you need to return Account here, but if so try this:
```suggestion
        Account account = new Account(number, balance)
        accountsList.add(account);
        return account;
```